apps/nova/src/pages/Tutors.tsx

```diff
@@ -13,7 +14,11 @@ const Tutors: React.FC = () => {
       <PageTitle
         title={intl("tutors.title")}
         fetching={tutors.query.isFetching && !tutors.query.isLoading}
-        className="mb-6"
+        className={cn(
+          "mb-6",
+          { "mb-36": tutors.query.isLoading },
+          { "mb-28": tutors.query.isError }
```

Comment:
A comment outside the scope of this pull request. 

You can use one object. There is no need to have a seperate object for each condtion. 

```suggestion
          { 
              "mb-36": tutors.query.isLoading,
              "mb-28": tutors.query.isError 
          }
```